### ISSUE TITLE
task_list_url is broken in gtimelog 0.10.0 on Python 3

### ISSUE THREAD

**Issue by teemmate** [gtimelog/gtimelog#92]
On fresh Ubuntu 16.04 adding path to task_list_url in gtimelogrc I get this output:

```
/usr/lib/python3/dist-packages/gtimelog/main.py:28: PyGIWarning: Gdk was imported without specifying a version first. Use gi.require_version('Gdk', '3.0') before import to ensure that the right version gets loaded.
  from gi.repository import GObject, GLib, Gdk, Gio, Gtk, Pango

** (gtimelog:9086): WARNING **: Error retrieving accessibility bus address: org.freedesktop.DBus.Error.ServiceUnknown: The name org.a11y.Bus was not provided by any .service files
/usr/lib/python3/dist-packages/gtimelog/main.py:31: PyGIWarning: AppIndicator3 was imported without specifying a version first. Use gi.require_version('AppIndicator3', '0.1') before import to ensure that the right version gets loaded.
  from gi.repository import AppIndicator3
Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/gtimelog/main.py", line 1065, in do_command_line
    self.do_activate()
  File "/usr/lib/python3/dist-packages/gtimelog/main.py", line 1130, in do_activate
    self.main_window = MainWindow(timelog, settings, tasks)
  File "/usr/lib/python3/dist-packages/gtimelog/main.py", line 228, in __init__
    self._init_ui()
  File "/usr/lib/python3/dist-packages/gtimelog/main.py", line 307, in _init_ui
    self.tick(True)
  File "/usr/lib/python3/dist-packages/gtimelog/main.py", line 984, in tick
    if self.tasks.check_reload():
  File "/usr/lib/python3/dist-packages/gtimelog/timelog.py", line 1034, in check_reload
    self.download()
  File "/usr/lib/python3/dist-packages/gtimelog/timelog.py", line 1043, in download
    urllib.urlretrieve(self.url, self.filename)
AttributeError: module 'urllib' has no attribute 'urlretrieve'
v-danek@danek-dexp:~$ gtimelog 
/usr/lib/python3/dist-packages/gtimelog/main.py:28: PyGIWarning: Gdk was imported without specifying a version first. Use gi.require_version('Gdk', '3.0') before import to ensure that the right version gets loaded.
  from gi.repository import GObject, GLib, Gdk, Gio, Gtk, Pango

** (gtimelog:9125): WARNING **: Error retrieving accessibility bus address: org.freedesktop.DBus.Error.ServiceUnknown: The name org.a11y.Bus was not provided by any .service files
/usr/lib/python3/dist-packages/gtimelog/main.py:31: PyGIWarning: AppIndicator3 was imported without specifying a version first. Use gi.require_version('AppIndicator3', '0.1') before import to ensure that the right version gets loaded.
  from gi.repository import AppIndicator3
Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/gtimelog/main.py", line 1065, in do_command_line
    self.do_activate()
  File "/usr/lib/python3/dist-packages/gtimelog/main.py", line 1130, in do_activate
    self.main_window = MainWindow(timelog, settings, tasks)
  File "/usr/lib/python3/dist-packages/gtimelog/main.py", line 228, in __init__
    self._init_ui()
  File "/usr/lib/python3/dist-packages/gtimelog/main.py", line 307, in _init_ui
    self.tick(True)
  File "/usr/lib/python3/dist-packages/gtimelog/main.py", line 984, in tick
    if self.tasks.check_reload():
  File "/usr/lib/python3/dist-packages/gtimelog/timelog.py", line 1034, in check_reload
    self.download()
  File "/usr/lib/python3/dist-packages/gtimelog/timelog.py", line 1043, in download
    urllib.urlretrieve(self.url, self.filename)
AttributeError: module 'urllib' has no attribute 'urlretrieve'

```

My gtimelog version is v0.10.0 as it says in Help>About

**Comment by mgedmin** [comment]
I've released version v0.10.1 to the Python Package Index: https://pypi.python.org/pypi/gtimelog.

You should be able to test it by installing with `pip install --user gtimelog` and running in a terminal with `~/.local/bin/gtimelog`.

It would be nice to build some fixed packages in a PPA, but packaging isn't my strong side.  :(

**Comment by menesis** [comment]
I have uploaded 0.10.1 version to the PPA https://launchpad.net/~gtimelog-dev/+archive/ubuntu/ppa/
